tools/omero/omero_filter.xml

```diff
@@ -42,10 +42,8 @@
                 </param>
             </when>
             <when value="KP">
-                <param name="value1" type="text" label="Key Value to search among the image IDs">
-                    <validator type="regex" message="Enter a valid Key to search in the OMERO server">^[\w\-. ]+$</validator>
-                </param>
-                <param name="value2" type="hidden" label="Pair Values to search among images IDs"/>
+                <param name="value1" type="text" label="Key Value to search among the image IDs"/>
```

Comment:
Should leave the validator (and add one to value2).

Should be labels for
- `value1` be `"Key to search among the image IDs"`
- `value2` be `"Value to search among the image IDs"`

Maybe rename `value1` to `key` and `value2` to `value`, for clarity?